# Request
string msg
string subject_id
---
# Response
bool success